### PR TITLE
Update documentation for alert-expandable

### PR DESCRIPTION
**Pull request by andresriveratoro**
Part of https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1011

### CODE REVIEW

File: src/_components/alert-expandable.md [hunk 1]
```diff
@@ -13,13 +13,38 @@ anchors:
...
+
+### Error alert
+
+Used when there is a problem or something destructive is about to occur.
+
+{% include storybook-preview.html story="components-va-alert-expandable--error" link_text="va-alert-expandable" %}
+
+### Continue alert
+
+{% include storybook-preview.html story="components-va-alert-expandable--continue" link_text="va-alert-expandable" %}
```
Comment (andresriveratoro) [inline src/_components/alert-expandable.md:43]: Do we know the use case for a continue alert?

Comment (humancompanion-usds) [reply src/_components/alert-expandable.md:43]: I think we may need to reach out to the team that created this component. I'm not sure what the use case would be for this component with the lock icon. It does seem specific enough that they probably had something in mind.

**Comment by humancompanion-usds** [comment]
We need more guidance here. Once you add the Storybook examples you can send this my way. I'll just add it to the queue of other things we're doing around Alerts. There are some use cases of this in [Facilities](https://www.va.gov/north-texas-health-care/locations/plano-va-clinic/) that are useful which I'd like to highlight as examples in this documentation.

**Comment by andresriveratoro** [comment]
> We need more guidance here. Once you add the Storybook examples you can send this my way. I'll just add it to the queue of other things we're doing around Alerts. There are some use cases of this in [Facilities](https://www.va.gov/north-texas-health-care/locations/plano-va-clinic/) that are useful which I'd like to highlight as examples in this documentation.

The Storybook examples are in. 👍

**Pull request merged**